init_configure_commands_tut.md

```diff
@@ -0,0 +1,156 @@
+<a href="https://aws-amplify.github.io/" target="_blank">
+    <img src="https://s3.amazonaws.com/aws-mobile-hub-images/aws-amplify-logo.png" alt="AWS Amplify" width="550" >
+</a>
+
+# AWS Amplify CLI initialization & configuration tutorial
+
+The AWS Amplify CLI helps you set up an AWS IAM user, set up an AWS profile on your local system and the initialize Amplify powered apps with all your defined configurations. We'll also walk through steps as to how you could share the Amplify generated backend configuration between other frontend projects and re-use the same backend infrastructure in-between multiple platforms/users.
+
+In this tutorial, we'll go walk through setting up an Amplify enabled project and go in detail as to what happens behind the scenes when doing so.
+
+# Command usage
+
+## amplify configure
+```bash
+$ amplify configure
+```
+The `amplify configure` command is a one-time setup step which helps you with the following:
```

Comment:
I would change to this: "The amplify configure command is a one-time setup step. After you perform this on your system you only need to run amplify init going forward on new projects. If you already have the AWS CLI configured on your system you can usually skip this step. The amplify configure step helps you with the following:"